api/searchCases/index.ts

```diff
@@ -1,157 +1,204 @@
+              },
+            };
+          } else {
+            match = {
+              wildcard: {
+                [field]: `*${searchTerm.toLowerCase()}*`,
+              },
+            };
+          }
+        } else {
+          match = {
+            match: {
+              [field]: {
+                operator: 'and',
+                query: searchTerm,
+              },
+            },
+          };
+        }
+        matchList.push(match);
+      }
     }
+  }
+
+  nativeEsQuery = {
+    from,
+    query: {
+      bool: {
+        must: matchList,
+      },
+    },
+    size,
+    sort,
+  };
+
+  return {
+    native_es_query: nativeEsQuery,
+    supplementary_data: ['*'],
+  };
 }
 
 function prepareSort(params) {
-    const sortQuery = []
-    if (params.hasOwnProperty('column') && params.hasOwnProperty('order') && params.hasOwnProperty('type')) {
-        let columnName: string
-
-        if (params.column.indexOf('[') === -1 ) {
-            columnName = `data.${params.column}${isKeywordSuffixNeeded(params.column, params.type)}`
-        } else {
-            const mappedName = fieldNameMapper(
-                params.column.replace('[', '').replace(']', '').toLowerCase(),
-                caseMetaDataFiledsMapping
-            )
-            columnName = `${mappedName}${isKeywordSuffixNeeded(mappedName, params.type)}`
-        }
-        const orderDirection = params.order === 0 ? 'ASC' : 'DESC'
-        sortQuery.push(
-            {
-                [columnName]: orderDirection,
-            }
-        )
+  const sortQuery = [];
+  if (params.hasOwnProperty('column') && params.hasOwnProperty('order') && params.hasOwnProperty('type')) {
+    let columnName: string;
+
+    if (params.column.indexOf('[') === -1) {
+      columnName = `data.${params.column}${isKeywordSuffixNeeded(params.column, params.type)}`;
+    } else {
+      const mappedName = fieldNameMapper(
+        params.column.replace('[', '').replace(']', '').toLowerCase(),
+        caseMetaDataFiledsMapping
+      );
+      columnName = `${mappedName}${isKeywordSuffixNeeded(mappedName, params.type)}`;
     }
-    return sortQuery
+    const orderDirection = params.order === 0 ? 'ASC' : 'DESC';
+    sortQuery.push(
+      {
+        [columnName]: orderDirection,
+      }
+    );
+  }
+  return sortQuery;
 }
 
 function isKeywordSuffixNeeded(columnName, type): string {
-    const types = ['Text', 'TextArea', 'FixedList', 'FixedListEdit', 'MultiSelectList', 'FixedRadioList', 'DynamicList']
-    const specialFields = ['reference', 'jurisdiction', 'state', 'case_type_id']
-    const isText = types.includes(type) || specialFields.includes(columnName)
-    return isText ? '.keyword' : ''
+  const types = ['Text', 'TextArea', 'FixedList', 'FixedListEdit', 'MultiSelectList', 'FixedRadioList', 'DynamicList'];
+  const specialFields = ['reference', 'jurisdiction', 'state', 'case_type_id'];
+  const isText = types.includes(type) || specialFields.includes(columnName);
```

Comment:
@paul-graham on a code review I did for you recently you used indexOf() because of IE11 support instead of this.

https://caniuse.com/?search=includes

Is that still valid?

Reply:
Oh! An excellent spot! Completely missed that one!

I think it is, yes. I believe we still need to support IE11...